lib/apiservers/portlayer/restapi/handlers/interaction_handlers.go

```diff
@@ -238,8 +291,9 @@ func (d *FlushingReader) AddFlusher(flusher GenericFlusher) {
 func (d *FlushingReader) WriteTo(w io.Writer) (written int64, err error) {
 	buf := make([]byte, 64)
 
+	nr, er := d.readDetectInit(buf)
```

Comment:
Modified this so that we only check for init bytes if this reader is created with init bytes.  Otherwise, it uses normal d.read.